moped-editor/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "atd-moped-editor",
   "author": "ATD Data & Technology Services",
   "license": "CC0 1.0 Universal",
-  "version": "1.2.0",
+  "version": "1.1.1",
```

Comment:
@johnclary, is this change intended as a patch? Right now its PR'ed against `main` so I think the version number should stay "1.2.0" for the next release. Otherwise we should PR against `production` for a patch right?

Reply:
John and I got a chance to talk about this yesterday in the dev sync. He indicated that he will put together a PR to get this merged into production. We also spoke about how I had flipped this diff in my mind, thinking that he was bumping the main branch to 1.2.0, which was an oversight on my part.

@johnclary - Since you're out of town, I hope you don't mind if I push a commit to this branch that nullifies this change to maintain main at 1.2.0.